OpenIPSL/Examples/Tutorial/Example_4/PfData/LoadData/LoadTemplate.mo

```diff
@@ -0,0 +1,8 @@
+within OpenIPSL.Examples.Tutorial.Example_4.PfData.LoadData;
+partial record LoadTemplate
+  "Record template for power flow solutions in loads"
+
+parameter OpenIPSL.Types.ActivePower PL1 "Load: load" annotation(Dialog(enable = false));
+parameter OpenIPSL.Types.ReactivePower QL1 "Load: load" annotation(Dialog(enable = false));
+
```

Comment:
The reason you see these "comments" in the power flow records is because they were automatically generated by the python scripts and functions described in [this paper](https://ecp.ep.liu.se/index.php/modelica/article/view/190/150).
Notice that this is also why the original names had "_"s. I do not see myself modifying the whole scripts in the short-term so, I will just remove all those confusing "comment" in the same way I renamed the records with underscores. I believe Sergio Dorado's original idea was to somehow link the record parameters to the OpenIPSL components. That's why I also added the "Load" prefix to the comment to follow a Class: instance convention. So, for the example above "Load" is the component type from OpenIPSL and "load" is the object name (component name). But anyway, I will remove the comments in all the power flow records.